terraform/lang/security/ebs-unencrypted-volume.yaml

```diff
@@ -1,20 +1,18 @@
 rules:
 - id: unencrypted-ebs-volume
   languages:
-  - generic
+  - hcl
   message: >-
     An EBS volume is configured without encryption enabled.
-  paths:
-    include:
-    - '*.tf'
   patterns:
-  - pattern: resource "aws_ebs_volume" "..."
+  - pattern: resource
```

Comment:
I'm surprised this works ...

Reply:
I was surprise too :D good surprise though